umbraco-cloud/upgrades/version-specific-upgrades/migrate-from-umbraco-7-to-8.md

```diff
@@ -0,0 +1,161 @@
+---
+description: This article will provide detailed steps on how to migrate a Umbraco 7 Cloud project to Umbraco 8.
+---
+
+# Migrate from Umbraco 7 to Umbraco 8 on Umbraco Cloud
+
+Taking your Umbraco CMS project from Umbraco 7 to 8 is called a migration as it requires that the data is migrated in the process. This article covers each step involved in this process.
+
+Read the [general article about Content migration](https://docs.umbraco.com/umbraco-cms/fundamentals/setup/upgrading/version-specific/migrate-content-to-umbraco-8.md#what-are-the-limitations) to learn more about limitations and other things related to migrating your Umbraco site from 7 to 8.
+
+## Video tutorial
+
+<iframe width="800" height="450" title="Migrate an Umbraco Cloud project from 7 to 8: Introduction" src="https://www.youtube.com/embed/wNIDdgdAt8s?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+You can find the full playlist here: [Migrate an Umbraco Cloud project from 7 to 8](https://www.youtube.com/playlist?list=PLgX62vUaGZsGceCXveMkprlDV5a3K94db)
```

Comment:
The videos in this playlist are not in order.

Reply:
This has now been sorted 💪